keadm/cmd/keadm/app/cmd/util/parser_test.go

```diff
@@ -0,0 +1,637 @@
+package strvals
```

Comment:
1. Fix the package to `util`;
2. Add a KubeEdge license;
3. Rename this file that add the suffix `_test` to the being tested go file(e.g. edgecore_strvals.go, edgecore_strvals_test.go);